### PR TITLE
support for embedded sql with python

### PR DESCRIPTION
**Pull request by sciabarracom**
this pm implements in pythonic way the redisqlite API

### CODE REVIEW

**Comment by rabbah** [comment]
The programming interface the sdk exposes for MySQL vs sql lite is supposed to be the same? That is if I change the client can I use the sql methods between the sql implementations interchangeably?

**Comment by sciabarracom** [comment]
The api "esql" (I renamed it on purpose) can be adapted easily to work with mysql, postgresl, oracle and everything but the underlying sql would be different so it is not so easy to make it portable. More or less like JDBC, same API but the code does not work from Oracle to Postgresql

File: README.md [hunk 1]
```diff
@@ -94,6 +94,104 @@ class StorageFile():
     def signed_url(version: str, action: str, expires: int, contentType: str) -> str:
 ```
 
+#### Embedded SQL support
```
Comment (jthomas) [inline README.md:97]: Can we move this content to "docs" and have a short section here like we did in the Node.js example?

**Review by jthomas — CHANGES_REQUESTED** [review]
I'm not the best Python person but looks good to me apart from the doc change.

**Comment by jthomas** [comment]
Any comments @rabbah before I merge? Speak now or forever hold your peace...

**Comment by rabbah** [comment]
Are the APIs on par between python and node? That is are we providing the same SDK?

**Comment by sciabarracom** [comment]
Yes the API are the same in nodejs and python

**Comment by rabbah** [comment]
👍

**Review by jthomas — APPROVED** [review]

**Pull request merged**